javascript/javascript1/week3/activity_tracker.js

```diff
@@ -0,0 +1,74 @@
+const activities = [];
+
+// Adding an activity
+function addActivity(date, activity, duration) {
+    let activityObject = {};
+    activityObject.date = date; //new Date().toLocaleDateString()
+    activityObject.activity = activity;
+    activityObject.duration = duration;
+    activities.push(activityObject);
+}
+
+addActivity("10.11.2020", "Youtube", 30);
+addActivity("10.11.2020", "Facebook", 40);
+addActivity("11.11.2020", "Youtube", 20);
+addActivity("12.11.2020", "Youtube", 20);
+console.log(activities);
+
+// Show my status
+// + Usage limit
+const myLimit = 60;
+function showStatus() {
+    let totalSpentTime = 0;
+    let status = "";
+    if(activities.length === 0) {
+        status = "Add some activities before calling showStatus";
+    } else {
+        for(let i = 0; i < activities.length; i++) {
+            for(const [key, value] of Object.entries(activities[i])) {
```

Comment:
It is great that you try a new thing, but do you really need this for here? 
I think it is more readble like : 
for (activity of activities) {
    totalSpentTime += activity.duration;
  }

Reply:
I made it unnecessarily complex. Changing code...